outdated/modules/httpd.reb

```diff
@@ -56,15 +73,13 @@ sys/make-scheme [
                         dispatch client
                     ]
 
-                    true [
-                        read client
-                    ]
+                    default [read client]
```

Comment:
Stylistic change.  Please [use DEFAULT](https://forum.rebol.info/t/default-now-usable-in-case-switch/739) instead of /ELSE or TRUE or other truthy things in cases.